client/src/util/promiseLimit.ts

```diff
@@ -68,35 +86,25 @@ export default class PromiseLimit {
   Private below
   **/
 
-  // @ts-expect-error ts-migrate(7006) FIXME: Parameter 'priority' implicitly has an 'any' type.
-  // eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types --- FIXME: disabled temporarily on migrate to TS.
-  _push(priority, fn, args) {
-    // eslint-disable-next-line @typescript-eslint/no-explicit-any --- FIXME: disabled temporarily on migrate to TS.
-    const order = (this as any).insertCount;
-    // eslint-disable-next-line @typescript-eslint/no-explicit-any --- FIXME: disabled temporarily on migrate to TS.
-    (this as any).insertCount += 1;
```

Comment:
insertCount didn't exist, changed to insertCounter

Reply:
good catch!!!